core/shared/src/main/scala/fs2/time/time.scala

```diff
@@ -46,7 +48,7 @@ package object time {
           val cancel = scheduler.scheduleAtFixedRate(d) {
             if (running.compareAndSet(false, true)) {
               val d = FiniteDuration(System.nanoTime, NANOSECONDS) - t0
-              F.unsafeRunAsync(signal.set(d))(_ => IO(running.set(false)))
+              Concurrent.unsafeRunAsync(signal.set(d))(_ => IO(running.set(false)))
```

Comment:
Yeah, sort of -- the `runAsync` method on `cats.effect.Effect` returns an `IO[Unit]` that kicks off the evaluation of `F`. https://github.com/typelevel/cats-effect/blob/master/core/shared/src/main/scala/cats/effect/Effect.scala#L35